docs/pages/2020_BIGMAP/sections/qe.rst

```diff
@@ -2,3 +2,279 @@
+
+.. code-block:: console
+
+    $ verdi calcjob inputls <PK>     # Shows the list of input files
+    $ verdi calcjob inputcat <PK>    # Shows the input file of the calculation
+    $ verdi calcjob outputls <PK>    # Shows the list of output files
+    $ verdi calcjob outputcat <PK>   # Shows the output file of the calculation
+    $ verdi calcjob res <PK>         # Shows the parser results of the calculation
+
+**Exercise:** A few questions you could answer using these commands (optional):
+
+    * How many atoms did the structure contain? How many electrons?
+    * How many k-points were specified? How many k-points were actually computed? Why?
+    * How many SCF iterations were needed for convergence?
+    * How long did Quantum ESPRESSO actually run (wall time)?
+
+From calculations to workflows
+------------------------------
+
+AiiDA can help you run individual calculations but it is really designed to help you run workflows that involve several calculations, while automatically keeping track of the provenance for full reproducibility.
+
+As the final step, we are going to launch the ``PwBandStructure`` workflow of the ``aiida-quantumespresso`` plugin.
+
+.. literalinclude:: include/snippets/demo_bands.py
+
+Download the :download:`demo_bands.py <include/snippets/demo_bands.py>` snippet and run it using
+
+.. code-block:: console
+
+    $ verdi run demo_bands.py
+
+This workflow will:
+
+    #. Determine the primitive cell of the input structure
+    #. Run a calculation on the primitive cell to relax both the cell and the atomic positions (``vc-relax``)
+    #. Refine the symmetry of the relaxed structure, and find a standardised primitive cell using SeeK-path_
+    #. Run a self-consistent field calculation on the refined structure
+    #. Run a band structure calculation at fixed Kohn-Sham potential along a standard path between high-symmetry k-points determined by SeeK-path_
+
+The workflow uses the PBE exchange-correlation functional with suitable pseudopotentials and energy cutoffs from the `SSSP library version 1.1 <https://www.materialscloud.org/discover/sssp/table/efficiency>`_.
+
+.. _SeeK-path: https://www.materialscloud.org/work/tools/seekpath
+
+.. K-point mesh is selected to have a minimum k-point density of 0.2 Å-1
+   A Marzari-Vanderbilt smearing of 0.02 Ry is used for the electronic occupations
+
+The workflow should take ~10 minutes on your virtual machine.
+You may notice that ``verdi process list`` now shows more than one entry.
+While you wait for the workflow to complete, let's start exploring its provenance.
+
+The full provenance graph obtained from ``verdi node graph generate`` will already be rather complex (you can try!), so let's try browsing the provenance interactively instead.
+
+Start the AiiDA REST API:
+
+.. code-block:: console
+
+    $ verdi restapi
+
+and open the |provenance browser| (from the browser inside the virtual machine).
```

Comment:
How would they open a browser in the VM's? Maybe this was originally planned for the QM but we need to adapt this (I think there should be a way to redirect the ports through the ssh connection).

Reply:
No clue! I just left this here since I couldn't test it earlier. I'll see if I can get it to work, else I'm sure @ltalirz can come to the rescue. 😅

Reply:
Perhaps the tutorial where this was copied from was run inside a QM desktop edition?
In any case, if they run on an AWS VM, they should certainly open this in a browser on their laptop

Reply:
Thanks @ltalirz! I'm going to test it and adjust the text today.

Reply:
I'm trying to run the RestAPI from the AWS with `verdi restapi`, and when opening the page in a browser the following error appears at the top in red:

```
Error while connecting to new REST endpoint! [Network Connection Error]
```

This is what I have on the AWS machine:

```console
$ verdi restapi
 * REST API running on http://127.0.0.1:5000/api/v4
 * Serving Flask app "aiida.restapi.run_api" (lazy loading)
 * Environment: production
   WARNING: This is a development server. Do not use it in a production deployment.
   Use a production WSGI server instead.
 * Debug mode: off
 * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
127.0.0.1 - - [25/Nov/2020 13:53:18] "GET /api/v3/server HTTP/1.1" 404 -
```
Reloading the page gives me more `404` lines. @ltalirz any idea what's going on here?

Reply:
you seem to be using /v3? `/api/v3/server`

Reply:
Of course, it was that simple 🤦‍♂️ .

Thanks @ltalirz, now it works like a charm! I still don't understand this RestAPI magic ✨, but it's pretty cool how easy it is to load!